core/soong_gen.go

```diff
@@ -274,7 +274,7 @@ func (gc *generateCommon) getHostBinModule(mctx android.TopDownMutatorContext) (
 }
 
 func (gc *generateCommon) createGenrule(mctx android.TopDownMutatorContext,
-	out []string, depfile string) {
+	out []string, depfile *bool) {
```

Comment:
Could this be a simple `bool` type rather than `*bool`? We aren't using this value as a module property, so we don't need to use the obfuscated types required for properties :)